src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -3457,4 +3457,7 @@ In certain complex graphs or host program scenarios, Automatic mode may cause in
   <data name="ExportSettingsFailedMessage" xml:space="preserve">
     <value>Export Failed</value>
   </data>
+  <data name="SplashScreenImportSettingsTooltip" xml:space="preserve">
+    <value>You can import custom settings here, which will overwrite your current settings. If you\'d like to preserve a copy of your current settings, export them before importing new settings. Settings not shown in the Preferences panel will be applied once Dynamo and any host program restarts.</value>
+  </data>
```

Comment:
@filipeotero Isn't this tooltip already defined? Can we reuse just in case in the future we only need to update at a single place

Reply:
Ok, I will find the current tooltip text resource